src/table/TableBodyWrapper.jsx

```diff
@@ -5,16 +5,28 @@ import TableBody from '@material-ui/core/TableBody';
 import TableRow from '@material-ui/core/TableRow';
 import { addSelectionListeners, reducer } from './selections-utils';
 import getCellRenderer from './cells/renderer';
-import { getBodyStyle } from './styling-utils';
+import { getBodyStyle, STYLING_DEFAULTS } from './styling-utils';
 import { bodyHandleKeyPress } from './cells/handle-key-press';
 
 const useStyles = makeStyles({
+  cellBase: {
+    '& td': {
+      color: ({ color }) => color,
+      fontSize: ({ fontSize }) => fontSize,
+      padding: ({ padding }) => padding,
+      height: STYLING_DEFAULTS.HEIGHT,
+      lineHeight: STYLING_DEFAULTS.HEAD_LINE_HEIGHT,
+      '&&:focus': {
+        boxShadow: STYLING_DEFAULTS.FOCUS_OUTLINE,
+      },
+    },
+  },
   hoverTableRow: {
     '&&:hover': {
       backgroundColor: 'rgba(0, 0, 0, 0)',
       '& td:not(.selected)': {
-        backgroundColor: ({ hoverBackgroundColor }) => hoverBackgroundColor,
-        color: ({ hoverFontColor }) => hoverFontColor,
+        backgroundColor: ({ hoverBackgroundColor }) => `${hoverBackgroundColor} !important`,
+        color: ({ hoverFontColor }) => `${hoverFontColor} !important`,
```

Comment:
Since inline styling always has higher priority than classes and this is on row level, we have to use !important here.